android/vcmi-app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="server_name">VCMI Server</string>
+    <string name="shortcut_play">Play VCMI</string>
```

Comment:
Can you add mention of this file in translation docs?

Reply:
Is already there: https://github.com/vcmi/vcmi/blob/d0ee1c89fbb1ace1f3d14ee616bdc89cb401d565/docs/translators/Translations.md?plain=1#L130